src/ValueObject/ImageMediaValueObject.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_theme\ValueObject;
+
+use Drupal\image\Plugin\Field\FieldType\ImageItem;
+
+/**
+ * Handle information about a media item.
+ */
+class ImageMediaValueObject extends MediaValueObject {
+
+  /**
+   * The alt of the image.
+   *
+   * @var string
+   */
+  protected $alt;
+
+  /**
+   * The parameter 'responsive' of the image.
+   *
+   * @var bool
+   */
+  protected $responsive;
+
+  /**
+   * ImageType constructor.
+   *
+   * @param string $src
+   *   Media URL, including Drupal schema if internal.
+   * @param string $alt
+   *   Image alt text.
+   * @param string $name
+   *   Name of the image, e.g. "example.jpg".
+   * @param bool $responsive
+   *   Responsiveness of the image.
+   */
+  public function __construct(string $src, string $alt = '', string $name = '', bool $responsive = TRUE) {
```

Comment:
Constructors on value objects should be set to `private`.

Reply:
:+1: